misc/pytorch_toolkit/chest_xray_screening/chest_xray_screening/utils/download_weights.py

```diff
@@ -1,17 +1,29 @@
-from google_drive_downloader import GoogleDriveDownloader as gdd
 from .get_config import get_config
+import os
+import zipfile
 
 def download_checkpoint():
-    config = get_config(action = 'gdrive')
-    gdd.download_file_from_google_drive(file_id=config['densenet']['file_id'],
-                                    dest_path=config['densenet']['dest_path'],
-                                    unzip=False)
-    gdd.download_file_from_google_drive(file_id=config['densenet_eff']['file_id'],
-                                    dest_path=config['densenet_eff']['dest_path'],
-                                    unzip=False)
+    config = get_config(action = 'download')
+    if not os.path.exists('model_weights'):
+        os.makedirs('model_weights')
+    densenet_url = config['densenet']['url']
+    densenet_path = config['densenet']['dest_path']
+    denseneteff_url = config['densenet_eff']['url']
+    denseneteff_path = config['densenet_eff']['dest_path']
+    os.system(f'wget -mnv {densenet_url} -O {densenet_path}')
```

Comment:
Please, do not use os.system, this is not safe. Use special python libraries instead - wget (which is already in your requirements.txt, requests or urllib.

Reply:
Okay, I Will make the change. I migrated from python wget to os.system as the latter was used [here](https://github.com/openvinotoolkit/training_extensions/blob/962b324d55595fa6be1ac285468933c706a58a0c/misc/pytorch_toolkit/text_recognition/text_recognition/utils/common.py#L30) in the text_recognition model.

Reply:
Well, thank you for this, I will need to change it also.

Reply:
Replaced `os.system` with pycurl. Python wget sometimes downloads an HTML file instead of the actual file.